src/checkers/community.js

```diff
@@ -97,3 +117,38 @@ async function community (github, opts) {
 
   return toCheck
 }
+
+function find (heading, opts, callback) {
```

Comment:
Test this function? Not sure how easy that would be - can you export it and unit test it?